x/ibc-account/types/account.go

```diff
@@ -13,27 +11,23 @@ import (
 	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
 )
 
+const (
+	IcaPrefix string = "ics27-1-"
```

Comment:
@milosevic Every registered port is prefixed with this. I was thinking it is a good way to track what version of ics27 an account was registered under (this may be useful for migrating accounts in the future, or even just to help with port collisions)